src/plugins/intel_cpu/src/nodes_factory.cpp

```diff
@@ -183,6 +184,7 @@ Node::NodesFactory::NodesFactory() : Factory("NodesFactory") {
     INTEL_CPU_NODE(Unique, Type::Unique);
     INTEL_CPU_NODE(Ngram, Type::Ngram);
     INTEL_CPU_NODE(RoPE, Type::RoPE);
+    INTEL_CPU_NODE(intel_cpu::node::VNode, Type::VNode);
```

Comment:
The CI error maybe caused by `intel_cpu::node::VNode`, just change it to `VNode`

Reply:
Renamed `VNode` into `BigPattern`